server/main.py

```diff
@@ -31,6 +31,7 @@ class Payload(BaseModel):
     path_length: float
     simulate_slit: bool
     mode: str
+    databank:str
```

Comment:
```suggestion
    databank: str
```
Out-of-scope: would be nice to add a check for black formatting with something like `black --check server/`